src/XperienceCommunity.Localization.Base/Localizer/XperienceStringLocalizer.cs

```diff
@@ -0,0 +1,133 @@
+﻿using CMS.ContentEngine;
+using CMS.DataEngine;
+using CMS.Helpers;
+using CMS.Websites.Routing;
+using Microsoft.Extensions.Localization;
+using System.Data;
```

Comment:
Can you remove this unused directive?

Reply:
Removed, thought i got those...

Reply:
Wait, those aren't unused, those are needed.